### PR TITLE
Improve trip segmentation

### PR DESCRIPTION
**Pull request by paultranvan**
This introduces many small things, mainly aiming at improving trip detection and segmentation. In short:
- Add a `STOP_TRACKING` tracking, which is very useful for the server. This one is the main improvement and actually fixes 3 things:
  - Wrong starting point date, somehow based on previous trip
  - Wrong start point, based on previous trip
  - Wrong trips, not corresponding to anything
- Adjust spatial frequency tracking
- Force location fetching for first detected motion activity, to improve starting point accuracy
- Avoid creating user at each upload
- Do not start tracking when already started: this was probably causing unexpected side-effects
- Increase batch size, w.r.t the proxy limit (15MB)
- Remove extra logs
- Increase duration for log and local data retention

### CODE REVIEW

File: src/app/domain/geolocation/tracking/user.js [hunk 1]
```diff
@@ -42,6 +42,30 @@ export const createUser = async user => {
...
+    })
+    if (respFetch.status === 403) {
+      return createUser(user)
+    }
+    response = await respFetch.json()
+    const uuid = response?.user_id?.['$uuid']
+    if (!uuid) {
+      await createUser(user)
+    }
+    return uuid
```
Comment (zatteo) [inline src/app/domain/geolocation/tracking/user.js:63]: S'il n'y a pas d'uuid, on créé un user mais on retourne quand même l'uuid inexistant ?

Comment (paultranvan) [reply src/app/domain/geolocation/tracking/user.js:63]: dans les faits ça ne change pas grand chose parce qu'on ne fait rien de cet uuid, mais c'est vrai qu'on devrait faire ça plus proprement, bien vu

File: src/app/domain/geolocation/tracking/tracking.js [hunk 2]
```diff
@@ -402,25 +404,43 @@ export const filterNonHeadingPointsAfterStillActivity = (
...
-    Log('No activity found in local storage. Get it from locations')
-    // Fallback when no activity was captured
-    // TODO: filter stationary
-    const activitiesFromLoc = getActivitiesFromLocations(locations)
-    return activitiesFromLoc
+  const savedActivities = await getActivities({ beforeTs })
+  let activitiesFromLocations = []
+  if (locations && locations.length > 0) {
+    activitiesFromLocations = locations.map(loc => {
+      const location = { ...loc }
```
Comment (zatteo) [inline src/app/domain/geolocation/tracking/tracking.js:469]: What is the purpose of cloning the object here ?

Comment (paultranvan) [reply src/app/domain/geolocation/tracking/tracking.js:469]: To avoid mutating the original object, as doing so can be sometimes tedious

**Review by zatteo — APPROVED** [review]

**Pull request merged**